src/cluster.c

```diff
@@ -107,6 +108,33 @@ ConnectionType *connTypeOfCluster(void) {
     return connectionTypeTcp();
 }
 
+void delUnOwnedKeys(void) {
+    if (!server.cluster_enabled) {
+        return;
+    }
+    unsigned int deleted_keys = 0;
```

Comment:
Is `unsigned int` sufficient to store total key count?

Reply:
outdated code but yes, the existing function uses unsigned int